docs/sources/style-guide/voice-tone-guidelines/index.md

```diff
@@ -29,7 +29,9 @@ Write in a natural and conversational style with language that reflects a positi
 
 A casual tone is accepted in UI copy and supporting material compared to traditional business writing.
 
-As long as they allow for shorter sentences, you can use the words _and_, _but_, and _so_ in moderation.
+As long as they allow for shorter sentences, you can use the words _and_, _but_, and _so_ in moderation. 
+
+Use ontractions like _isn't_ for _is not_ to help to convey an informal tone. But [be careful](https://www.grammarly.com/blog/contractions/).
```

Comment:
Seems like this line duplicate https://github.com/grafana/writers-toolkit/pull/209/files#diff-d0f37c5e0037e4a48703cdc766fdfd3c91ef47f1c67e693c09900864dddfd829R61. I recommend moving the link to line 61 and removing the rest from here.

Reply:
I don't think a little duplication is bad in this case. The use of contractions is both a style issue and a tonal issue. People learn better sometimes with a little repetition.

Reply:
Perhaps "Remember that it's okay to use contractions like _isn't_ for _is not_ to help to convey a conversational tone..."?